### DISCUSSION TITLE
I completed my first mind map for school and would appreciate advice on improving it. I greatly appreciate it!

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1465] [Help]
Hey everyone! I am new to using mind maps as a study tool. Since I am new to this technique, I want to learn from and be critiqued by those with more experience. My current mind map is linked below. This mind map includes pictures, but I am not they will appear with this file.

https://www.dropbox.com/scl/fi/j13ks2saj5t0b7qy82fnx/Community-Property-Midterm-Reduced.mm?rlkey=9j9ax0lsfsijfu2cqykwn99a6&dl=0

**Comment by euu2021** [comment]
If it works for you, then it's perfect.

For me, I find the colors by levels confusing. I prefer to color code according to specific meaning, and deal with the levels by keeping the node symmetrical (all with the same horizontal size). Also, I like to breakdown nodes a bit more. 

For example, rewriting your "Two ways to rebut general CP presumption" node, I would do like this (I'm not sure if I understood the content correctly):

![image](https://github.com/freeplane/freeplane/assets/77707706/3a80cd88-03f3-4c15-8408-1f1e3bcca319)

The orange color shows to me that the two nodes are sibling specimens of the same category. The red color shows that this node is an opposing idea to the previous one.

About the "Symmetric Nodes", see this video:  https://www.youtube.com/watch?v=L4GYa0QuhPY
About "Splitting information" see this video: https://www.youtube.com/watch?v=PlSmJoMv44g

**Reply by adamfreeplaneuser01** [reply]
Thank you for taking the time to respond! I want to get better at making my maps more symmetric. I watched that video that you sent and now I'm trying to figure out how to make all my nodes the same size so that they will be symmetric. 
Here is another example from another mind map of mine where it shows I am doing a poor job with symmetry lol:
![Screenshot 2023-10-14 121216](https://github.com/freeplane/freeplane/assets/147225942/5fec3c0a-03a9-4d8f-b758-c6d0dcf3c2f6)

**Reply by adamfreeplaneuser01** [reply]
Edit: I think I was about to figure out how to make all the nodes the same width. I just set the default min width to 7cm and the max width to 7 cm. 

I guess my only other question would be about splitting up nodes. 
![Screenshot 2023-10-14 170616](https://github.com/freeplane/freeplane/assets/147225942/b3bcf1ec-f636-4b75-b1c4-81a4ab93992b)

I know you might not be familiar with these topics so I'll try to explain. The first three nodes are elements that make up a rule. And then the first child node that says "not comparable to DF's hands" is a comment that goes with the first element of the rule. Then the summary node is there to show what the result will be if all the elements are satisfied. Do you think this is a good way to go about this? And do you think that it's a good idea for me to use "blank nodes" to help keep everything symmetric? I circled where the two blank nodes currently are, along with the "comment" that goes along with the first element of the rule.

**Reply by euu2021** [reply]
> The first three nodes are elements that make up a rule. And then the first child node that says "not comparable to DF's hands" is a comment that goes with the first element of the rule. Then the summary node is there to show what the result will be if all the elements are satisfied. Do you think this is a good way to go about this?

Yes, I think it makes sense, to me. 

> And do you think that it's a good idea for me to use "blank nodes" to help keep everything symmetric?

Yes, I like this. I even use a conditional style specially for that.
Later, I will create another thread explaining how to do this.

**Reply by euu2021** [reply]
> And do you think that it's a good idea for me to use "blank nodes" to help keep everything symmetric?

I talked a bit about "blank nodes " in #1470

**Reply by adamfreeplaneuser01** [reply]
Thanks, this is awesome!

**Comment by dpolivaev** [comment]
Freeplane standard templates contain a template called "BigMap.mm". It differs from the rest. It configures all nodes to have the same size and even has edge colors depending on the distance to the root. You can try it and use it for inspiration.

**Reply by adamfreeplaneuser01** [reply]
Thanks! How do I change the map's template to this template?

**Reply by adamfreeplaneuser01** [reply]
Edit: never mind, I figured that part out. The only thing I need to figure out now is how to change the default text size for all node levels at once

**Reply by euu2021** [reply]
> Edit: never mind, I figured that part out. The only thing I need to figure out now is how to change the default text size for all node levels at once

Work with Styles: https://docs.freeplane.org/user-documentation/styles.html?highlight=style#styles
